src/format.ts

```diff
@@ -244,20 +244,23 @@ const queryParamsDimension: Dimension = {
       r.result.queryString || ansi.format('[gray]{<none>}'),
 };
 
-const implementationDimension: Dimension = {
-  label: 'Impl',
-  format: (r: ResultStats) => r.result.implementation,
-};
-
 const versionDimension: Dimension = {
   label: 'Version',
-  format: (r: ResultStats) => r.result.version,
+  format: (r: ResultStats) => r.result.version || ansi.format('[gray]{<none>}'),
 };
 
 const browserDimension: Dimension = {
   label: 'Browser',
-  format: (r: ResultStats) =>
-      `${r.result.browser.name}\n${r.result.browser.version}`,
+  format: (r: ResultStats) => {
+    if (r.result.userAgent === '') {
+      // This happens with remote URLs, since we don't get a chance to
+      // instrument any requests to find the user agent.
+      // TODO We could make one no-op request just to probe the user agent.
```

Comment:
webdriver can also execute JS to retrieve the UA

Reply:
Huh cool, I didn't know https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/userAgent existed. Will do later.